content-services/latest/config/mtls.md

```diff
@@ -0,0 +1,283 @@
+---
+title: mTLS
+---
+
+This section describes a recommended approach for generating secure keys and setting up certificates. It is not required that you use this approach if you have an alternative solution that you already use.
+
+If you're installing Content Services using the distribution zip, you need to generate certificates for the repository and Solr. By default, the distribution zips are configured to use SSL, so you'll need to generate these certificates to get your system to run successfully.

```

Comment:
Theoretically solr isn't mandatory, but I don't know if there is a distribution zip that doesn't use it. I'll leave a note here that I can follow-up on later.